src/main/java/org/synyx/urlaubsverwaltung/application/application/ApplicationForLeaveDetailsViewController.java

```diff
@@ -390,9 +404,21 @@ private void prepareDetailView(Application application, int year, String action,
             model.addAttribute(BEFORE_APRIL_ATTRIBUTE, DateUtil.isBeforeApril(LocalDate.now(clock), acc.getYear()));
         }
 
+        // Signed in person is allowed to manage
+        final boolean isDepartmentHead = departmentService.isDepartmentHeadAllowedToManagePerson(signedInUser, application.getPerson());
+        model.addAttribute("isDepartmentHead", isDepartmentHead);
+        final boolean isSecondStageAuthority = departmentService.isSecondStageAuthorityAllowedToManagePerson(signedInUser, application.getPerson());
+        model.addAttribute("isSecondStageAuthority", isSecondStageAuthority);
+        model.addAttribute("isBoss", signedInUser.hasRole(BOSS));
+        model.addAttribute("isOffice", signedInUser.hasRole(OFFICE));
```

Comment:
finally 🤩🥳

Reply:
ich würde glaube ich sogar so weit gehen, hier die Sachlichkeit als booleans zu setzen. z. B. `permittedToAllow`, `permittedToRefer`, ... (dann ist die view fast komplett dumm und rendert nur noch.)

hatte mal den Gedanken vielleicht sogar die links zu setzen im `Model`. und in der view dann einfach zu prüfen ob der link existiert oder nicht.